### DISCUSSION TITLE
Тесты производительности

### DISCUSSION THREAD

**Discussion by SerhiiZahuba** [mikopbx/Core#79] [General ( Обо всём )]
Проводились ли тесты производительности системы ?
Пример под колл центр, одновременно будет порядка 450 звонков.

Есть ли возможность бросать входящие звонки на разные очереди в зависимости от ответа который даст стороння база данных ? Пример поступает звонок - идёт проверка номера в базе - если он есть то бросаем на очередь номер один если нет то очередь 2.

**Comment by boffart** [comment]
Нет, таких тестов не проводили. 

> Есть ли возможность бросать входящие звонки на разные очереди в зависимости от ответа который даст стороння база данных

В базовом функционале не предусмотрено. Нужно разрабатывать этот функционал. Похоже на наш модуль умной маршрутизации для 1С.
